inst/include/RcppSimdJson/deserialize/simplify.hpp

```diff
@@ -18,13 +18,11 @@ template <Type_Policy type_policy, utils::Int64_R_Type int64_opt, Simplify_To si
 inline SEXP
 simplify_list(simdjson::dom::array array, SEXP empty_array, SEXP empty_object, SEXP single_null) {
     Rcpp::List out(r_length(array));
-
     auto i = R_xlen_t(0);
-    for (auto&& element : array) {
+    for (auto element : array) {
```

Comment:
Notice here how I removed the moves. There is no use moving because our API only generates lightweight instances.